File: java/ABTestExample/app/src/main/java/com/pubscale/sdkone/example/abtestexample/event_listener/RewardedAdEventListener.java
package com.pubscale.sdkone.example.abtestexample.event_listener;

public interface RewardedAdEventListener {
    void onReward();
    void onAdLoading();
    void onAdLoaded();
    void onAdLoadFailed();
    void onAdShowFailed();
    void onAdOpened();
    void onAdClosed();
}

File: kotlin/ABTestExample/app/src/main/java/com/pubscale/sdkone/example/abtestexample/event_listener/RewardedAdEventListener.kt
package com.pubscale.sdkone.example.abtestexample.event_listener

interface RewardedAdEventListener {
    fun onReward()
    fun onAdLoading()
    fun onAdLoaded()
    fun onAdLoadFailed()
    fun onAdShowFailed()
    fun onAdOpened()
    fun onAdClosed()
}